ext.gradle

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2017, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ *
+ * This file lists versions of most of the dependencies, used in this sample application.
+ *
+ * As long as module dependencies are periodically updated, the version-update
+ * commits often pollute the VCS log for `build.gradle` file. Often, such commits interfere
+ * with the important changes to the build procedure.
+ *
+ * Having a distinct configuration file like this one allows to trace dependency version changes
+ * and other configuration updates separately.
+ */
+
+project.ext {
+
+    // Application version.
+    //
+    appVersion = "0.0.1-SNAPSHOT"
+
+    // Versions of Spine modules.
+    //
+    spineVersion = '0.10.4-SNAPSHOT'
+    spineBaseVersion = '0.10.0'
+
+    // Versions of the most definitive third-party libraries.
+    //
+    protobufGradlePluginVersion = '0.8.3'
+    guavaVersion = '20.0'
```

Comment:
If we stick with Java 8, we can use [a later version of Guava](https://github.com/google/guava/releases).

Reply:
As discussed vocally, while Spine allows JDK 7, the examples/quickstart apps are going to remain compatible with it.

Also, for now any tests are omitted, as we don't know any preferences of our users. Depending on JDK version and on taste it could be anything from JUnit 4 to the latest TestNG. 

So to keep this project a _quick_ start I'd refrain from setting any controversial defaults.